src/main/java/Duke.java

```diff
@@ -1,10 +1,53 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        String dukeLogo =  " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String convStart = dukeLogo + "Hello! I'm Duke\n" + "What can I do for you?";
+        String convEnd= "Bye. Hope to see you again soon!";
+
+        ListDuke list = new ListDuke();
+
+        printOutput(convStart);
+
+        String lineInput;
```

Comment:
"Variables should be initialized where they are declared and they should be declared in the smallest scope possible." - this String could be initialized and declared in the while loop right below, a slightly smaller scope than where it is here.